faqs/gtn/get_data_upload_tool_using_FTP.md

```diff
@@ -0,0 +1,18 @@
+---
+title: How do I load data using FTP?
+area: Loading data
+layout: faq
+box_type: tip
+contributors: [jennaj, bernandez]
+---
+
+* Load the data using line command FTP or a client.
```

Comment:
like in the other FAQs, please make sure that there are clear and concise steps that are easy to follow

Reply:
(also, remember the numbering)